Dockerfile

```diff
@@ -36,5 +22,39 @@ RUN wget --quiet https://chromedriver.storage.googleapis.com/89.0.4389.23/chrome
   && mv $PWD/chromedriver /usr/local/bin \
   && rm -f chromedriver_linux64.zip
 
+# Copy node install from node-image
+COPY --from=node-image /usr/local/bin/node /usr/local/bin/
+COPY --from=node-image /usr/local/lib/node_modules /usr/local/lib/node_modules
+RUN ln -s ../lib/node_modules/npm/bin/npm-cli.js /usr/local/bin/npm
+RUN ln -s ../lib/node_modules/npm/bin/npx-cli.js /usr/local/bin/npx
+
+RUN rm -rf /var/lib/apt/lists/*
```

Comment:
You could make this a single `RUN` step, so it wouldn't create too many docker layers..

Reply:
Hmm:

> Where possible, only copy the artifacts you need into the final image. This allows you to include extra tools and/or to debug information in your intermediate build stages, without increasing the size of the final image.

Ideally it'd be nice to have a version with tons of layers for working on the image and then completely flatten it for the actual version. Maybe this is too much trouble though.

Reply:
Seems like --squash does this flattening.

Reply:
Yeah, but that means that you can't reuse existing layer (e.g. from base image). So better to just avoid creating too many layer in the Docker image. Also because this might be built and used by other people.

Reply:
Unfortunately it requires an experimental docker-daemon flag to enable it, so perhaps we can wait until they release it as a normal feature. For now, will merge these steps as you suggested.

Reply:
The Dockerfile API is pretty clumsy.

Reply:
Is there a way to merge these COPYs:
```
COPY --from=node-image /usr/local/bin/node /usr/local/bin/
COPY --from=node-image /usr/local/lib/node_modules /usr/local/lib/node_modules
```